tests/database/odbc-base/Makefile

```diff
@@ -0,0 +1,42 @@
+include ../../common.mk
+
+PROG=odbc_app
+# PROG_SRC=$(PROG).c
+IMAGE_SIZE=50M
+
+EXECUTION_TIMEOUT=120
+SGXLKL_VERBOSE=0 
+SGXLKL_ENV=SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=1 SGXLKL_GETTIME_VDSO=0 SGXLKL_TAP=sgxlkl_tap0 SGXLKL_HOST_IMPORT_ENV="DB_SERVER_NAME,DB_NAME,DB_PASSWORD,DB_USERID"
+SGXLKL_HW_PARAMS=--hw-debug
+SGXLKL_SW_PARAMS=--sw-debug
+
+SGXLKL_ROOTFS=sgx-lkl-rootfs.img
+
+.DELETE_ON_ERROR:
+.PHONY: all clean
+
+# $(SGXLKL_ROOTFS): $(PROG_SRC)
```

Comment:
Ideally, this should list all the source files that are copied into the Docker image. Otherwise, remove this line.